cvmfs/CMakeLists.txt

```diff
@@ -59,6 +61,7 @@ set (CVMFS_CLIENT_SOURCES
   history_sqlite.cc history_sqlite.h
   json_document.cc json_document.h
   loader.h
+  kvstore.h kvstore.cc
```

Comment:
This was my mistake, I think.  The kvstore files should be before `loader.h` for alphabetical order.